packages/matter.js/src/cluster/definitions/OnOffCluster.ts

```diff
@@ -0,0 +1,321 @@
+     */
+    Toggle = 2
+}
+
+export const enum OnOffEffectIdentifier {
+    DelayedAllOff = 0,
+    DyingLight = 1
+}
+
+/**
+ * Input to the OnOff offWithEffect command
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_1} § 1.5.7.4
+ */
+export const TlvOffWithEffectRequest = TlvObject({
+    /**
+     * The EffectIdentifier field specifies the fading effect to use when turning the device off. This field shall
+     * contain one of the non-reserved values listed in Values of the EffectIdentifier Field of the OffWithEffect
+     * Command.
+     *
+     * @see {@link MatterApplicationClusterSpecificationV1_1} § 1.5.7.4.1
+     */
+    effectIdentifier: TlvField(0, TlvEnum<OnOffEffectIdentifier>()),
+
+    /**
+     * The EffectVariant field is used to indicate which variant of the effect, indicated in the EffectIdentifier
+     * field, SHOULD be triggered. If the server does not support the given variant, it shall use the default variant.
+     * This field is dependent on the value of the EffectIdentifier field and shall contain one of the non-reserved
+     * values listed in Values of the EffectVariant Field of the OffWithEffect Command.
+     *
+     * @see {@link MatterApplicationClusterSpecificationV1_1} § 1.5.7.4.2
+     */
+    effectVariant: TlvField(1, TlvUInt8)
+});
+
+/**
+ * Bit definitions for TlvOnOffControl
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_1} § 1.5.7.6.1
+ */
+export const OnOffControlBits = { acceptOnlyWhenOn: BitFlag(0), reserved: BitFlag(1) };
+
+/**
+ * The value of OnWithTimedOff.onOffControl
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_1} § 1.5.7.6.1
+ */
+export const TlvOnOffControl = TlvBitmap(TlvUInt8, OnOffControlBits);
+
+/**
+ * Input to the OnOff onWithTimedOff command
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_1} § 1.5.7.6
+ */
+export const TlvOnWithTimedOffRequest = TlvObject({
+    /**
+     * The OnOffControl field contains information on how the server is to be operated. This field shall be formatted
+     * as illustrated in Format of the OnOffControl Field of the OnWithTimedOff Command.
+     *
+     * The AcceptOnlyWhenOn sub-field is 1 bit in length and specifies whether the OnWithTimedOff command is to be
+     * processed unconditionally or only when the OnOff attribute is equal to TRUE. If this sub-field is set to 1, the
+     * OnWithTimedOff command shall only be accepted if the OnOff attribute is equal to TRUE. If this sub-field is set
+     * to 0, the OnWithTimedOff command shall be processed unconditionally.
+     *
+     * @see {@link MatterApplicationClusterSpecificationV1_1} § 1.5.7.6.1
+     */
+    onOffControl: TlvField(0, TlvOnOffControl),
+
+    /**
+     * The OnTime field is used to adjust the value of the OnTime attribute.
+     *
+     * @see {@link MatterApplicationClusterSpecificationV1_1} § 1.5.7.6.2
+     */
+    onTime: TlvField(1, TlvNullable(TlvUInt16)),
```

Comment:
{ min: 0, max: 254 }

Reply:
Where does that come from?  Spec says "all" though there's a lot of prose in that chapter so maybe I'm missing it

Reply:
ha, all fine, seems to be a fix from invalid xml or specs. also can not find it anymore. ok